pyxnat/tests/resources_test.py

```diff
@@ -290,11 +292,11 @@ def test_get_zip():
         assert os.path.exists(f)
 
 def test_project_aliases():
-    project = central.select('/project/nosetests')
-    assert project.aliases() == ['nosetests2']
+    project = central.select('/project/nosetests3')
+    assert project.aliases() == ['nosetests32']
```

Comment:
Involuntary project name modification after a find&replace run (completely harmless nevertheless)?

Reply:
This one would be correct :wink: (project `nosetests` is aliased to `nosetests32`).